src/Chapter13/Chapter13.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <ChapterNumber>13</ChapterNumber>
+    <LangVersion>10.0</LangVersion>
```

Comment:
IS there a reason we are setting this at the project level? We have a Directory.Build.props file that has a single version that applies to all projects.